### PR TITLE
LPS-133438 | master

### PR DESCRIPTION
**Pull request by david-gutierrez-mesa**
https://issues.liferay.com/browse/LPS-133438
cc @Tim-Cao

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/macros/asset/assetlist/AssetListsAdmin.macro [hunk 1]
```diff
@@ -1,5 +1,41 @@
...
+
+		for (var categoryName : list "${categoryNames}") {
+			Portlet.selectTreeNode(nodeName = "${categoryName}");
+		}
+
+		SelectFrameTop();
+
+		AssertClick(
+			locator1 = "Button#DONE",
+			value1 = "Done");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/asset/assetlist/AssetListsAdmin.macro:34]:
```suggestion
			AssetCategorization.addMultipleCategories(
				categoryNameList = "${categoryNameList}",
				vocabularyName = "${vocabularyName}");
```

File: portal-web/test/functional/com/liferay/portalweb/macros/asset/assetlist/AssetListsAdmin.macro [hunk 2]
```diff
@@ -312,6 +348,22 @@ definition {
...
+				value1 = "${categoryName}");
+
+			var key_tagName = "${categoryName}";
+
+			Click(locator1 = "AssetCategorization#TAG_REMOVE");
+
+			AssertElementNotPresent(locator1 = "AssetCategorization#CATEGORY");
+		}
+	}
+
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/asset/assetlist/AssetListsAdmin.macro:366]: Should move to AssetCategorization.macro

Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/macros/asset/assetlist/AssetListsAdmin.macro:366]:
```suggestion
	macro removeCategories {
		for (var categoryName : list "${categoryNameList}") {
		        var key_categoryName = "${categoryName}";

			AssertTextEquals(
				locator1 = "AssetCategorization#CATEGORY",
				value1 = "${categoryName}");

			var key_tagName = "${categoryName}";

			Click(locator1 = "AssetCategorization#TAG_REMOVE");

			AssertElementNotPresent(locator1 = "AssetCategorization#CATEGORY");
		}
	}

```

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/CollectionCategoryFilter.testcase [hunk 3]
```diff
@@ -2024,4 +2024,95 @@ definition {
...
+		}
+
+		task ("Add a web content article with the Global categories") {
+			var n = "1";
+
+			for (var vocabularyName : list "Topic,Audience") {
+				var categoryId = JSONCategory.getCategoryId(
+					categoryName = "Category Name ${n}",
+					groupName = "Global",
+					vocabularyName = "${vocabularyName}");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/CollectionCategoryFilter.testcase:2050]: Redundant step

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/CollectionCategoryFilter.testcase [hunk 4]
```diff
@@ -2024,4 +2024,95 @@ definition {
...
+	@priority = "5"
+	test FilterResultsOfDynamicCollectionByCategoryFromGlobalSite {
+		task ("Add a category in Topic, Audience") {
+			var n = "1";
+
+			for (var vocabularyName : list "Topic,Audience") {
+				JSONCategory.addCategory(
+					groupName = "Global",
+					title = "Category Name ${n}",
+					vocabularyName = "${vocabularyName}");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/CollectionCategoryFilter.testcase:2037]:
```suggestion
				var categoryId = JSONCategory.addCategory(
					groupName = "Global",
					title = "Category Name ${n}",
					vocabularyName = "${vocabularyName}");
```

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/CollectionCategoryFilter.testcase [hunk 5]
```diff
@@ -2024,4 +2024,95 @@ definition {
...
+
+			AssetListsAdmin.editAssetList(assetListTitle = "Dynamic Collection");
+
+			AssetListsAdmin.addCategoriesFilter(
+				categoryNames = "Category Name 1",
+				queryAndOperator = "Any",
+				queryContains = "Contains");
+		}
+
+		task ("Assert the selected web content article is viewable from the View Content action") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/CollectionCategoryFilter.testcase:2080]: View collection items

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/CollectionCategoryFilter.testcase [hunk 6]
```diff
@@ -2024,4 +2024,95 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-131309. Collection Dynamic Filter should return the items of Internal Categories from Global Site."
+	@priority = "5"
+	test FilterResultsOfDynamicCollectionByCategoryFromGlobalSite {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/CollectionCategoryFilter.testcase:2029]: FilterCollectionItemsByGlobalCategory

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/CollectionCategoryFilter.testcase [hunk 7]
```diff
@@ -2024,4 +2024,95 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-131309. Collection Dynamic Filter should return the items of Internal Categories from Global Site."
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/fragment/outoftheboxfragments/CollectionCategoryFilter.testcase:2027]: The user could filter collection items by global category.

**Comment by Tim-Cao** [comment]
Hi @ozysouza, could you move the case to AssetListsUseCase.testcase ?
The Collection Category Filter is a fragment only available to collection page but unrelated to this case.

**Comment by ozysouza** [comment]
@ Hi @Tim-Cao I did the changes that you requested, but it didn't come to the pull request. May you know what happened? 
on the git's log doesn't had any warning.

`$ git push origin LPS-133438 -f
Enumerating objects: 56, done.
Counting objects: 100% (56/56), done.
Delta compression using up to 12 threads
Compressing objects: 100% (28/28), done.
Writing objects: 100% (38/38), 3.58 KiB | 1.79 MiB/s, done.
Total 38 (delta 21), reused 0 (delta 0), pack-reused 0
remote: Resolving deltas: 100% (21/21), completed with 13 local objects.
To github.com:ozysouza/liferay-portal.git
 + 9eda5ae...2906371   LPS-133438 -> LPS-133438 (forced update)`

**Comment by Tim-Cao** [comment]
Hi @ozysouza, this PR is sent to David but sent by his remote branch so you cannot force pushed. The user can only force pushed original branch not others remote branch. 
Anyway, I could fetch your remote branch then work on it since you already update your remote branch.

**Comment by Tim-Cao** [comment]
See https://github.com/david-gutierrez-mesa/liferay-portal/pull/171

**Comment by Tim-Cao** [comment]
ci:close

File: portal-web/test/functional/com/liferay/portalweb/macros/asset/assetlist/AssetListsAdmin.macro (hunk 1 above)
Comment (Tim-Cao) [reply portal-web/test/functional/com/liferay/portalweb/macros/asset/assetlist/AssetListsAdmin.macro:34]: Sorry, I revert my change after double checked

**Pull request closed without merging**